rfc/github-cdevents.md

```diff
@@ -0,0 +1,34 @@
+| dev.cdevents.change.updated     | pull_request                | This event occurs when there is activity on a pull request. <details><summary>Actions</summary> {"Action type":{"assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "enqueued", <br/>"archived", "labeled", "locked", "milestoned", "ready_for_review","reopened", "review_request_removed", "review_requested", "synchronized", "unassigned", "unlabeled", "unlocked"}</br> |
+| dev.cdevents.change.reviewed    | pull_request_review_comment | This event occurs when there is activity on a pull request review comment. <details><summary>Actions</summary> {"Action type":{"created", "reviewed", "commented"}                                                                                                                                                                                                                                                                                                |
+| dev.cdevents.change.reviewed    | pull_request_review         | This event occurs when there is activity relating to a pull request review. <details><summary>Actions</summary> {"Action type":{"edited", "dismissed", "submitted"}  </details>                                                                                                                                                                                                                                                                                   |
+| dev.cdevents.change.modified    | pull_request_review_thread  | This event occurs when there is activity relating to a comment thread on a pull request. <details><summary>Actions</summary> {"Action type":{"resolved", "unresolved"}  </details>                                                                                                                                                                                                                                                                                |
+| dev.cdevents.artifact.modified  | package/registry_package    | This event occurs when there is activity relating to GitHub Packages/registry. <details><summary>Actions</summary> {"Action type":{"published", "updated"}  </details>                                                                                                                                                                                                                                                                                            |
```

Comment:
The `published` action should correspond to the `dev.cdevents.artifact.published` event instead.